.github/workflows/pre-commit.yml

```diff
@@ -0,0 +1,15 @@
+name: pre-commit
+
+on:
+  pull_request:
+  push:
+
+jobs:
+  pre-commit:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: 3.x
+      - uses: pre-commit/action@v3.0.0
```

Comment:
We could also set up https://pre-commit.ci/lite.html, and then it would provide fixes that people could automatically apply to their PRs.